content/blog/brulee-0-2-0/index.Rmd

```diff
@@ -0,0 +1,47 @@
+---
+output: hugodown::hugo_document
+
+slug: brulee-0-2-0
+title: brulee 0.2.0
+date: 2022-09-26
+author: Max Kuhn
+description: >
+    Version 0.2.0 of brulee introduces learning rate schedulers. 
+
+photo:
+  url: https://unsplash.com/photos/wiTWDYLURr8
+  author: Alex Munsell
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidymodels, brulee]
+---
+
+We're thrilled to announce the release of [brulee](https://tidymodels.github.io/brulee/) 0.2.0. brulee contains several basic modeling functions that use the torch package infrastructure, such as: neural networks, linear regression, logistic regression, and multinomial regression. 
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("brulee")
+```
+
+This blog post will describe the changes to the package. You can see a full list of changes in the [release notes](https://tidymodels.github.io/brulee/news/index.html)
+
+There were two main additions to brulee. 
+
+First, since brulee is focused on fitting models to _tabular data_, we've been be moving away from optimizing via stochastic gradient descent (SGD) as the default. For `brulee_mlp()`, we switched the default optimizer from SGD to more traditional quasi-newton methods, specifically to Broyden–Fletcher–Goldfarb–Shanno algorithm (BFGS) method. You can still use SGD via the `optimizer` option. 
```

Comment:
```suggestion
First, since brulee is focused on fitting models to _tabular data_, we have moved away from optimizing via stochastic gradient descent (SGD) as the default. For `brulee_mlp()`, we switched the default optimizer from SGD to more traditional quasi-newton methods, specifically to Broyden–Fletcher–Goldfarb–Shanno algorithm (BFGS) method. You can still use SGD via the `optimizer` option. 
```